test/bigtest/network/config.js

```diff
@@ -36,4 +38,6 @@ export default function config() {
 
   this.get('/data-export/mapping-profiles/:id', (schema, request) => schema.mappingProfiles.find(request.params.id).attrs);
   this.delete('/data-export/mapping-profiles/:id', {}, 204);
+
```

Comment:
I think we can remove this empty line.

Reply:
This whole config formatted that way, where different endpoints separated by empty lines, I don't think it's a good idea